apps/graphql/src/apps/booking/queries/ConfirmPayment.js

```diff
@@ -0,0 +1,29 @@
+// @flow
```

Comment:
maybe we should have a folder `/bookings/mutations` where this file should be. It is not a query, but a mutation right?